xcodeproj/internal/opts.bzl

```diff
@@ -501,19 +537,59 @@ under {}""".format(opt, package_bin_dir))
 
     return unhandled_opts
 
+def _process_user_swiftcopts(opts):
+    """Processes user-provided Swift compiler options.
+
+    Args:
+        opts: A `list` of Swift compiler options.
+
+    Returns:
```

Comment:
Can we add a comment in here that says for any processing that happens here we need ensure the values are properly filtered from processing in `_process_full_swiftcopts()`?